### PR TITLE
CLUT-4: Build simple list of stories screen.

### PR DESCRIPTION
**Pull request by Rieon**
very simple collection view for story.

### CODE REVIEW

File: viYarn/Story/view/StoryViewController.swift [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+//  Created by vieon on 14.09.2018.
+//  Copyright © 2018 RIEON. All rights reserved.
+//
+
+import UIKit
+
+class StoryViewController: UICollectionViewController, UICollectionViewDelegateFlowLayout{
+    
+    let cellID = "CellId"
+    var storyCollectionView: UICollectionView!
```
Comment (FlareKin) [inline viYarn/Story/view/StoryViewController.swift]: p0: can't spot a place where we use it. Please remove.

File: viYarn/Story/view/StoryViewController.swift [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
+        let screenSize = UIScreen.main.bounds
+        let cellWidth: CGFloat = screenSize.width * 0.9
+        let cellHeight: CGFloat = screenSize.height * 0.7
+        return CGSize(width: cellWidth, height: cellHeight)
```
Comment (FlareKin) [inline viYarn/Story/view/StoryViewController.swift]: p0: instead use view.bounds.height and view.bounds.width properties.

**Review by FlareKin — CHANGES_REQUESTED** [review]
Need a little polish. Everything else looks great

**Comment by Rieon** [comment]
fixed

**Review by FlareKin — APPROVED** [review]
LGTM

**Pull request merged**